index.html

```diff
@@ -0,0 +1,1729 @@
+                </span>
+              </span>
+
+              <h5>
+                <a href="#"
+                  >How Amazon's Bet on Autonomous Vehicles Can Help Protect Us
+                  from Viruses</a
+                >
+              </h5>
+            </div>
+
+            <div class="row">
+              <div class="m-item col-lg col-sm-12 p-2">
+                <div>
+                  <img
+                    src="resources/jair-bolsonaro-brazil-favelas-coronavirus-covid-19.jpg"
+                    alt=""
+                  />
+                </div>
+
+                <span class="category">
+                  <span class="mag-cat">periscope</span>
+                  <span class="cat">
+                    <a href="#">culture</a>
+                  </span>
+                </span>
+
+                <h5>
+                  <a href="#"
+                    >As Brazil's Bolsonaro Enjoys BBQ Despite COVID-19, the Poor
+                    Face Starvation</a
+                  >
+                </h5>
+              </div>
+
+              <div class="m-item col-lg col-sm-12 p-2">
+                <div>
+                  <img src="resources/cul-books-banner.jpg" alt="" />
+                </div>
+                <span class="category">
+                  <span class="mag-cat">downtime</span>
+                  <span class="cat">
+                    <a href="#">culture</a>
+                  </span>
+                </span>
+
+                <h5>
+                  <a href="#"
+                    >The 20 Must-Read Fiction and Nonfiction Books of the
+                    Summer</a
+                  >
+                </h5>
+              </div>
+
+              <div class="m-item col-lg col-sm-12 p-2">
+                <div>
+                  <img
+                    src="resources/cul-uncharted-frozentreats-banner.jpg"
+                    alt=""
+                  />
+                </div>
+
+                <span class="category">
+                  <span class="mag-cat">downtime</span>
+                  <span class="cat">
+                    <a href="#">culture</a>
+                  </span>
+                </span>
+
+                <h5><a href="#">World's Finest Frozen Treats</a></h5>
+              </div>
+
+              <div class="m-item col-lg col-sm-12 p-2">
+                <div class="picture">
+                  <img src="resources/unmasked-man.jpg" alt="" />
+                  <div class="f-badge">
+                    <a class="p-1" href="#">infocus</a>
+                  </div>
+                </div>
+
+                <h5><a href="#">UNMASKED MAN</a></h5>
+              </div>
+            </div>
+          </div>
+
+          <div class="editors container-fluid">
+            <div
+              class="magazine-title mb-2 p-3 border-bottom d-flex align-items-center justify-content-center"
+            >
+              <span class="icon"></span>
+              EDITOR'S PICK
```

Comment:
- [ ] Editor's pick should have two columns for medium devices.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83581141-92869580-a503-11ea-84c1-da44a4359dfa.png) | ![image](https://user-images.githubusercontent.com/57414789/83581123-86023d00-a503-11ea-8de5-d5cd12b3c7f1.png)